client/src/services/fetchReviews.js

```diff
@@ -0,0 +1,11 @@
+const fetchReviews = async (showId) => {
+  try {
+    const response = await fetch(`/api/v1/reviews/${showId}`)
```

Comment:
so close! This should be swapped to match standard convention.

Change this:
```
/api/v1/reviews/${showId}
```
to this:
```
/api/v1/shows/${showId}/reviews
```
By convention this conveys that we are grabbing all reviews for a specific show. 

It would also would be ideal to fetch to a `showReviews` router per https://learn.launchacademy.com/lessons/persisting-related-records-in-objection-and-express